config/manager/manager_configmap.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: manager
+  namespace: system
+data:
+  cluster_host_external: ""
```

Comment:
@mmulholla `primazactl`'s users should be able to set this value. As an example, in a two kind cluster environment, this should be set to the internal IP of the container that's running the kind cluster on which primaza is installed